twinkle-app/twinkle.bll.services/src/test/UserRegisterTest.java

```diff
@@ -0,0 +1,20 @@
+import org.testng.annotations.Test;
```

Comment:
Please add package declaration and the same folder structure.

```suggestion
package com.github.donkeyrit.twinkle.bll;

import org.testng.annotations.Test;
```

Reply:
Done